includes/database/engine/class-column.php

```diff
@@ -536,7 +505,7 @@ private function validate_args( $args = array() ) {
 			'zerofill'      => 'wp_validate_boolean',
 			'binary'        => 'wp_validate_boolean',
 			'allow_null'    => 'wp_validate_boolean',
-			'default'       => array( $this, 'sanitize_default' ),
```

Comment:
We need this back in order for `null` values to be saved properly. Right now they're getting converted to empty strings and that's breaking a bunch of the unit tests.

Reply:
Added this back.